tests/BuilderWithErrors/BuilderWithErrorsTest.php

```diff
@@ -7,18 +7,20 @@
 use Doctrine\RST\Builder;
 use Doctrine\Tests\RST\BaseBuilderTest;
 
-class BuilderWithErrors extends BaseBuilderTest
+class BuilderWithErrorsTest extends BaseBuilderTest
 {
     protected function configureBuilder(Builder $builder): void
     {
         $builder->getConfiguration()->abortOnError(false);
+        $builder->getConfiguration()->silentOnError(true);
     }
 
-    public function testMalformedTable(): void
+    public function testNoContentDirectiveError(): void
     {
-        $contents = $this->getFileContents($this->targetFile('index.html'));
-        self::assertStringContainsString('<table', $contents);
-        self::assertStringNotContainsString('<tr', $contents);
```

Comment:
There was aa test asserting that a broken table doesn't render... except that since aa long time ago, we ARE able to render (at least mostly) broken tables :).